lib/typescript/httpclient/model/Message.ts

```diff
@@ -28,13 +30,36 @@ export enum MessageState {
   delivered = 'DELIVERED',
 }
 
+export enum MessageSenderType {
```

Comment:
as we call this `SenderType` it may be worth reflecting the naming in the frontend. Especially because we intend to simplify in #572 soon enough and it would be easier to catch this